src/scss/variables-admin.scss

```diff
@@ -14,36 +14,38 @@
 // COLORS
 //=============================================================================
 
-$venetian-red:                               #dc0018;
-$red:                                        #f7001d;
-$mocassin:                                   #fffab2;
-$cerulean:                                   #006699;
-$pattens-blue:                               #d8e8ef;
-$malibu:                                     #66afe9;
-$solitude:                                   #e7edef;
-$clear-day:                                  #f2f7f9;
+$venetian-red: #dc0018;
+$red: #f7001d;
+$mocassin: #fffab2;
+$cerulean: #006699;
+$pattens-blue: #d8e8ef;
+$malibu: #66afe9;
+$solitude: #e7edef;
+$clear-day: #f2f7f9;
 
-$black:                                      #000000;
-$night-rider:                                #333333; // gray 80%;
-$coal:                                       #454545; // gray 73%;
-$empress:                                    #757575; // gray 54%;
-$silver:                                     #cccccc; // gray 20%;
-$light-grey:                                 #d5d5d5; // gray 16%;
-$gainsboro:                                  #e5e5e5; // gray 10%;
-$smoke:                                      #f5f5f5; // gray 4%;
-$white:                                      #ffffff;
+$black: #000000;
+$night-rider: #333333; // gray 80%;
+$coal: #454545; // gray 73%;
+$empress: #757575; // gray 54%;
+$silver: #cccccc; // gray 20%;
+$light-grey: #d5d5d5; // gray 16%;
+$gainsboro: #e5e5e5; // gray 10%;
+$smoke: #f5f5f5; // gray 4%;
+$white: #ffffff;
 
-$sherpa-blue:                                #03344d;
-$fire-engine-red:                            #bb1122;
-$cadillac:                                   #884488;
+$sherpa-blue: #03344d;
+$fire-engine-red: #bb1122;
+$cadillac: #884488;
 
 // TYPOGRAPHY
 //=============================================================================
 
 //** Fonts variables
-$frutiger:    'Frutiger Neue', Helvetica, Arial, sans-serif;
+$frutiger: 'Frutiger Neue', Helvetica, Arial, sans-serif;
 $admin-icons: 'Admin Icons';
 
 // MISC
 //=============================================================================
 $focus-outline: 3px solid $malibu;
+// Number of background layers (in addition to 'blank')
+$numberOfBackgrounds: 3;
```

Comment:
As no other file is using this, I would keep it in `MapFooterBackgroundSelector.vue`
but as you've correctly fixed it, maybe only declare it only once (at the top of the SCSS section)

Reply:
OK